client/src/main.ts

```diff
@@ -19,7 +19,7 @@ const app = createApp(App)
 const colourTheme = {
   colors: {
     primaryBlue: '#193855',
-    pink: '#D12974',
+    primaryPink: '#D12974',
     lightBlue: '#3FBEE0',
     creamWhite: '#E9DAC4',
     textGrey: '#4F4F4F',
```

Comment:
Sorry to keep requesting trivial changes, but could you change the name of ``textGrey``? The reason is that when we use this colour in the components, we will be doing something like ``<p class="text-textGrey">Hello</p>``. IMO it looks a little dumb to have "text" twice in a row like that. So can you change "text" to some other word? Some options could be "theme", "custom", or "primary".

Reply:
All good. I've changed it